### PR TITLE
Fix dependencies of Compute on KEyvault/Storage to be now on common

### PR DESCRIPTION
**Pull request by praries880**
Fix for issue #6436

### CODE REVIEW

**Comment by maddieclayton** [comment]
@praries880 To fix Jenkins, add an entry to Profile changelog.

**Comment by MiYanni** [comment]
@praries880 I fixed the merge conflicts from preview, and merged preview into your branch.

**Comment by praries880** [comment]
@azuresdkci test this please

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/1060/

File: src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageProvider.cs [hunk 1]
```diff
@@ -12,9 +12,9 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Management.Storage;
```
Comment (MiYanni) [inline src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageProvider.cs]: Common changes are not needed here. Should be done in common repo.

File: src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageService.cs [hunk 2]
```diff
@@ -12,6 +12,7 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Management.Storage.Version2017_10_01.Models;
```
Comment (MiYanni) [inline src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageService.cs]: Common changes are not needed here. Should be done in common repo.

File: src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1 [hunk 3]
```diff
@@ -584,24 +584,25 @@ function Get-Location
 {
     param([string]$providerNamespace, [string]$resourceType, [string]$preferredLocation)
     $provider = Get-AzureRmResourceProvider -ProviderNamespace $providerNamespace
-    $resourceTypes = $null
```
Comment (MiYanni) [inline src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1]: Common changes are not needed here. Should be done in common repo.

File: src/ResourceManager/Compute/Commands.Compute.Test/Common/ComputeTestController.cs [hunk 4]
```diff
@@ -26,9 +25,7 @@
 using Microsoft.Azure.Management.Network;
 using Microsoft.Azure.Management.KeyVault;
 #if NETSTANDARD
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/Common/ComputeTestController.cs]: Since the #if has no content now, you should just remove the #else and do `#if !NETSTANDARD`

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DscExtensionTests.ps1 [hunk 5]
```diff
@@ -103,10 +103,10 @@ function Test-GetAzureRmVMDscExtension
     finally
     {
 		# Cleanup
-		if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
-		{
+		#if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DscExtensionTests.ps1]: Commented code should be removed. Since the `finally` has no code, it should be removed too.

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1 [hunk 6]
```diff
@@ -113,10 +113,10 @@ function Test-SetAzureRmVMSqlServerAKVExtension
     finally
     {
         # Cleanup
-        if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
-        {
+        #if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1]: Commented code should be removed. Since the `finally` has no code, it should be removed too.

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/NewVhdVMTests.cs [hunk 7]
```diff
@@ -34,7 +34,7 @@ public NewVhdVMTests(Xunit.Abstractions.ITestOutputHelper output)
 #else
         [Fact(Skip = "Resources -> ResourceManager, needs re-recorded")]
 #endif
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.RunType, Category.CheckIn)]
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/NewVhdVMTests.cs]: Why was this changed?

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1 [hunk 8]
```diff
@@ -227,10 +227,10 @@ function Test-SetAzureRmVMSqlServerExtension
...
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1]: Commented code should be removed. Since the `finally` has no code, it should be removed too.

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1 [hunk 9]
```diff
@@ -351,10 +351,10 @@ function Test-SetAzureRmVMSqlServerExtensionWith2016Image
...
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1]: Commented code should be removed. Since the `finally` has no code, it should be removed too.

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineTests.ps1 [hunk 10]
```diff
@@ -130,11 +130,13 @@ function Test-VirtualMachine
         $a = $vm1 | Out-String;
         Write-Verbose("Get-AzureRmVM output:");
         Write-Verbose($a);
+		Assert-NotNull $a
```
Comment (MiYanni) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineTests.ps1]: Tabs make me sads. 😢

File: src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1 (hunk 3 above)
Comment (praries880) [reply src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1]: Thee changes are already present in common, I just wanted to keep the files in both the repos on the same page...

Comment (MiYanni) [reply src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1]: @praries880 These files will be deleted soon. I have to resolve https://github.com/Azure/azure-powershell/issues/6946 and then I'm going to delete these files. I'll hopefully have a PR next week.

Comment (praries880) [reply src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1]: Then it does not hurt to leave the changes as is does it? The changes are already there in the new repo.. n this file is not used anyhow.

Comment (MiYanni) [reply src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1]: After we created the common repo, we said we wouldn't take anymore changes here. The rationale being that it would only cause confusion to update anything in these directories, since these files are unused. If you just copy/paste the current version of the file in the repo to your fork, you'll effectively 'revert' it.

File: src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageProvider.cs (hunk 1 above)
Comment (praries880) [reply src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageProvider.cs]: done

**Comment by praries880** [comment]
new on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/1062/

**Comment by MiYanni** [comment]
On demand using appropriate workers: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/1070/

**Comment by MiYanni** [comment]
ps-sign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1138/
